ambry-store/src/main/java/com.github.ambry.store/CostBenefitInfo.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+/**
+ * CostBenefitInfo Stores cost benefit information for a potential candidate to be compacted
+ */
+class CostBenefitInfo implements Comparable<CostBenefitInfo> {
+  private final long cost;
+  private final int benefit;
+  private final Double costBenefitRatio;
+  private final String firstLogSegmentName;
+  private final String lastLogSegmentName;
+
+  CostBenefitInfo(String firstLogSegmentName, String lastLogSegmentName, long cost, int benefit) {
```

Comment:
Cost is total amount of data that needs to be copied in bytes(and hence long). Benefit is no of log segments reclaimed and hence an integer.